driver-core/src/test/resources/client-side-encryption/missingKey.json

```diff
@@ -61,12 +61,48 @@
         },
         "bsonType": "object"
     },
-    "key_vault_data": [],
+    "key_vault_data": [
```

Comment:
This test was passing for the wrong reason before. This should have been inserting into a different collection from the key vault namespace specified in the options.